src/middleware.ts

```diff
@@ -0,0 +1,42 @@
+import { sequence, defineMiddleware } from 'astro:middleware';
```

Comment:
Filename should indicate what this middleware does

Reply:
This file is actually the only middleware file allowed if I understood correctly.
The way we make more middleware's is by adding them to the sequence function in the bottom of the page.

Reply:
nvm, you can create a middleware/index.ts and then include files from there.